### ISSUE TITLE
@long-description for CSS files

### ISSUE THREAD

**Issue by aminomancer** [MrOtherGuy/fx-autoconfig#39]
I think because the whole metadata block needs to be wrapped in a block comment for CSS, the `@long-description` tag won't work since it requires a block comment of its own. So the end comment token for the inner description block ends the outer comment, and everything after it gets parsed as CSS.

```css
/* ==UserScript==
// @long-description
// @description
/*
long description here
*/
// ==/UserScript==
*/
```

Not sure what a good solution would be though. Maybe match for some other start/end token for long descriptions when it's a CSS file?

```css
/* ==UserScript==
// @long-description
// @description
<!--
long description here
-->
// ==/UserScript==
*/
```

**Comment by MrOtherGuy** [comment]
For now it should be enough to just open another block comment immediately after ending `@long-description` - but yeah it's a bit unfortunate that the style header needs to be different in the first place.

**Comment by aminomancer** [comment]
Ah I see what you mean. For some reason I thought the regex was looking for these block comment tokens, but I see now it's just the JS line comment tokens. So this only really matters for linting I suppose, but your solution works for that.

This way seems decent as far as aesthetics go:

```css
/* ==UserScript==
// @long-description
// @description
*//*
long description here
*//* 
// ==/UserScript==
*/
```

**Comment by MrOtherGuy** [comment]
Right, unfortunately I don't think we can really do much about this since the script header is semi-standard across all loader - except we extend it a bit. So even if we came up with some better header format it would then mean that scripts written for other loaders would not work properly.
